eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver.php

```diff
@@ -165,6 +171,10 @@ public function hasParameter($paramName, $namespace = null, $scope = null)
      */
     public function getParameter($paramName, $namespace = null, $scope = null)
     {
+        if (!$this->container instanceof ContainerBuilder && $this->siteAccess->matchingType === 'uninitialized') {
```

Comment:
Maybe for readability you could make this a method: `$this->checkContainerIsInitialized($paramName)`. It would allow to sandbox the logic and make it self documented.